ansible_collections/arista/cvp/docs/how-to/v3/cv_device_v3.md

```diff
@@ -35,7 +35,7 @@ CVP_DEVICES:
 
 #### Optional inputs
 
-- `state`: Define if module should `create`(default) or `delete` devices from CV
+- `state`: Define if module should `create`(default) or `delete` devices from CV. By default it is set to `present`.
```

Comment:
Would be good to improve that by referencing the 2 keywords supported (`present` and `factory_reset`)

Reply:
Updated accordingly